interfaces/seccomp/backend.go

```diff
@@ -101,20 +101,20 @@ func (b *Backend) deriveContent(spec *Specification, opts interfaces.Confinement
 			content = make(map[string]*osutil.FileState)
 		}
 		securityTag := hookInfo.SecurityTag()
-		addContent(securityTag, opts, spec.SnippetForTag(securityTag), content)
+		addContent(securityTag, true, opts, spec.SnippetForTag(securityTag), content)
 	}
 	for _, appInfo := range snapInfo.Apps {
 		if content == nil {
 			content = make(map[string]*osutil.FileState)
 		}
 		securityTag := appInfo.SecurityTag()
-		addContent(securityTag, opts, spec.SnippetForTag(securityTag), content)
+		addContent(securityTag, false, opts, spec.SnippetForTag(securityTag), content)
 	}
 
 	return content, nil
 }
 
-func addContent(securityTag string, opts interfaces.ConfinementOptions, snippetForTag string, content map[string]*osutil.FileState) {
+func addContent(securityTag string, isHook bool, opts interfaces.ConfinementOptions, snippetForTag string, content map[string]*osutil.FileState) {
```

Comment:
My personal pet-peeve: I don't like `bool` in a function signature. I much prefer flags, even if its just a single option for these flags. I think bool makes it harerd to read, e.g: `addContext(securityTag, false..)` vs `addContext(securityTag, isHook|needsJuice...)`.